man/adoc/bpftrace.adoc

```diff
@@ -978,6 +978,7 @@ These exist for the lifetime of `bpftrace` itself and can be accessed from all a
 Map names always start with a `@`, e.g. `@mymap`.
 
 All valid identifiers can be used as `name`.
+ Note that there are restrictions on the use of identifiers beyond C, e.g. `config` is reserved as a language keyword and may not be used directly.
```

Comment:
This seems to work for me:
```
sudo bpftrace -e 'BEGIN { @config = 1; exit(); }'
```

Reply:
I guess it depends on your read of "directly". `@config` is namespaced by `@`

Reply:
This is in a section about map naming so I think this addition is confusing as no identifiers without @ can be used for maps.

Reply:
Yeah, you're right... the problem only exists for bare identifiers, so you couldn't name a `struct config` or subprogram `config`. As this change isn't really changing any of those semantics, I'll drop this line here for now but might be worthwhile to have a kind of language spec/guide (both for bring forward some clear decisions and for the user). Happy to help on that.